mono/utils/mono-signal-handler.c

```diff
@@ -0,0 +1,69 @@
+#include "mono-signal-handler.h"
+
+#include <signal.h>
+#include <string.h>
+#include "glib.h"
+
+struct mono_sigpair
+{
+	int signo;
+	const char* signame;
+};
+
+struct mono_sigpair mono_signames[] =
+{
+	{SIGABRT, "SIGABRT"},
+	{SIGKILL, "SIGKILL"},
+	{SIGTRAP, "SIGTRAP"},
+	{SIGSYS, "SIGSYS"},
+	{SIGSEGV, "SIGSEGV"},
+	{SIGQUIT, "SIGQUIT"},
+	{SIGFPE, "SIGFPE"},
+	{SIGILL, "SIGILL"},
+	{SIGBUS, "SIGBUS"}
+};
+
+struct mono_sigpair *sigpair_buf = NULL;
+int sigpair_buflen;
```

Comment:
both of these could be `static`. (and  then you don't need to explicitly init `sigpair_buf` with `NULL`)